test/tests/functional/pbs_job_array.py

```diff
@@ -394,8 +397,9 @@ def test_job_array_history_duration(self):
         b = {'job_state': 'X'}
         self.server.expect(JOB, b, subjid_1)
         self.server.expect(JOB, a, subjid_2, attrop=PTL_AND)
-        # offset is 150 secs as server will purge db once in
-        # 2 mins + 30 sec of history duration
+        msg = "offset is 150 secs as server will purge db once"
+        msg += " in 2 mins plus 30 sec of history duration"
+        self.logger.info(msg)
```

Comment:
saying 'offset is' is fine, but I think it would read better if you said 'Waiting' instead.